### PR TITLE
ARE-6130: Python wrapper for currencies endpoint

### PR DESCRIPTION
**Pull request by almabekov**
Python method to wrap the /exchange_rate_tables/id/currencies/ endpoint (ticket for endpoint was ARE-6109).

### CODE REVIEW

File: analyzere/resources.py [hunk 1]
```diff
@@ -31,7 +31,15 @@ def profile(self):
 # Exchange rate tables
 
 class ExchangeRateTable(DataResource):
-    pass
+    def currencies(self):
+        path = '%s/currencies' % self._get_path(self.id)
```
Comment (mcgillij) [inline analyzere/resources.py]:
```suggestion
        path = '%s/currencies' % self._get_path(self.id)
        path = '{}/currencies'.format(self._get_path(self.id))
```
I think we generally use `.format` or you could use a new fancy `f`string.
would look something like path = f'{self._get_path(self.id)}/currencies'

Comment (almabekov) [reply analyzere/resources.py]: Updated as suggested.

File: tests/test_resources.py [hunk 2]
```diff
@@ -80,3 +80,43 @@ def test_get_results(self, reqmock):
...
+    # ARE-6130 wrapper for the exchange rate table unique currencies function
+    def test_unique_currencies(self, reqmock):
+        # mock for the Exchange Rate table request
+        reqmock.get('https://api/exchange_rate_tables/abc_id',
+                    status_code=200, text='{"id":"abc_id"}')
+
+        fx_table = ExchangeRateTable.retrieve('abc_id')
+        # mock for the currencies method call
+        reqmock.get('https://api/exchange_rate_tables/abc_id/currencies', status_code=200,
+                    text='{'
```
Comment (Fishflavored) [inline tests/test_resources.py]: Can you put the text part in one line? feels that will look a bit nicer.

File: tests/test_resources.py [hunk 3]
```diff
@@ -80,3 +80,43 @@ def test_get_results(self, reqmock):
...
+        assert currencies == {'EUR', 'CAD'}
+
+    # we still should have save the empty currencies list
+    def test_unique_currencies_empty(self, reqmock):
+        reqmock.get('https://api/exchange_rate_tables/abc_id',
+                    status_code=200, text='{"id":"abc_id"}')
+        fx_table = ExchangeRateTable.retrieve('abc_id')
+
+        reqmock.get('https://api/exchange_rate_tables/abc_id/currencies', status_code=200,
+                    text='{'
```
Comment (Fishflavored) [inline tests/test_resources.py]: Same here

File: tests/test_resources.py (hunk 2 above)
Comment (almabekov) [reply tests/test_resources.py]: Updated.

File: tests/test_resources.py (hunk 3 above)
Comment (almabekov) [reply tests/test_resources.py]: Updated.

**Pull request merged**